tdrs-backend/tdpservice/stts/migrations/0006_alter_stt_filenames.py

```diff
@@ -56,6 +16,4 @@ class Migration(migrations.Migration):
             name='filenames',
             field=models.JSONField(blank=True, max_length=512, null=True),
         ),
-        migrations.RunPython(add_filenames)
```

Comment:
If not removed, this would continue to cause hiccups with future updates to STT model.